### PR TITLE
N2WS Dashboards update

### PR DESCRIPTION
**Pull request by eliadeini**
### What does this PR do?

Update N2WS dashboards according to recommendation given by Sean Chow from DataDog during a meeting between N2WS team and DataDog team.

### Motivation

To make the dashboards look more professional. 

### Review checklist

- [ ] PR has a Dashboards fix (https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

File: n2ws/manifest.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json"
```
Comment (hithwen) [inline n2ws/manifest.json:29]: Dahsboard validation is failing with following errors:
```
Validating all Dashboard definition files...
n2ws...  FAILED
  assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json does not exist
n2ws...  FAILED
  assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json does not exist
15 valid files
2 invalid files
```
You can run this validation locally with `ddev validate dashboards`

**Comment by eliadeini** [comment]
Hi,
I’ve seen these errors, but the files are located on the correct paths in the repository.
The files are there, therefore I don’t understand why I get “does not exist” error.

Thanks,
Eliad

File: n2ws/manifest.json (hunk 1 above)
Comment (hithwen) [reply n2ws/manifest.json:29]: the file assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json does not exist, instead the n2ws/assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json does. Note that in one areas has caps A and in the other one is lowercase.
The same happens with GraphicalVersion.

Comment (eliadeini) [reply n2ws/manifest.json]: Hi, I tried to run `ddev validate dashboards` without adding "n2ws" to the path and it passed, failed with "n2ws".
In addition, I committed with "n2ws" and it failed. So the checks fails either way ...

**Comment by eliadeini** [comment]
Hi, I tried to run `ddev validate dashboards` without adding "n2ws" to the path and it passed, failed with "n2ws".
In addition, I committed with "n2ws" and it failed. So the checks fails either way ...

I renamed the file, so the capital A is OK.

File: n2ws/manifest.json (hunk 1 above)
Comment (hithwen) [reply n2ws/manifest.json]: If you're running this on windows or macos, that have case insensitive file systems the validation will pass

File: n2ws/manifest.json [hunk 2]
```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "n2ws/assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "n2ws/assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "n2ws/assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json"
```
Comment (hithwen) [inline n2ws/manifest.json:29]:
```suggestion
	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
```

File: n2ws/manifest.json [hunk 3]
```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "n2ws/assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "n2ws/assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json",
```
Comment (hithwen) [inline n2ws/manifest.json:28]:
```suggestion
	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json ",
```

File: n2ws/manifest.json [hunk 4]
```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
```
Comment (hithwen) [inline n2ws/manifest.json:29]: This file is still on the PR, did you mean to delete it?

**Review by hithwen — CHANGES_REQUESTED** [review]

File: n2ws/manifest.json (hunk 3 above)
Comment (hithwen) [reply n2ws/manifest.json:28]: You need to either apply this change or rename the file to "N2WSBackup&Recovery-GraphicalVersion.json" (notice the capitalisation difference)

File: n2ws/manifest.json (hunk 2 above)
Comment (hithwen) [reply n2ws/manifest.json:29]: You need to either apply this change or rename the file to "N2WSBackup&Recovery-GraphicalVersion-Areas.json" (notice the capitalisation difference)

**Comment by hithwen** [comment]
>  I renamed the file, so the capital A is OK.

It does not seem so

N2WS Dashboards update by eliadeini · Pull Request #756 · DataDog:integrations-extras 2020-11-10 12-44-21

**Comment by eliadeini** [comment]
I’m sorry, this directory is the GitHub directory I’m using to submit files to the PR:

[cid:image001.png@01D6B76F.75BB2460]
It looks like I’m submitting files with capitalized names, but GitHub changes the names … (This directory is synced with Git Hub).
Do you know why?

Thanks,
Eliad

**Review by hithwen — APPROVED** [review]

**Pull request merged**